.idea/copyright/rls.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (c) &amp;#36;today.year. REAL LIFE SCIENCES INC" />
```

Comment:
Does it really make sense to include a RLS copyright notice? I am not an Intellij user, so I don't know what it is used for, but I have the feeling that it could (should?) be removed.

Reply:
agree we can remove

Reply:
I don't know how this is used by IntelliJ, so this question is a bit hard to answer. If it is unusual to not include such a copyright statement, I would indeed suggest to replace the value with ARX's standard copyright notice ("Copyright (c) &amp;#36;today.year. Fabian Prasser and contributors"). If this is implemented, please also change the name of the file, e.g. to arx.xml, and insert a reasonable value into "<option name="myName" value="rls" /". Can we also include a reference to the Apache 2 license here?